canvas_modules/common-canvas/locales/common-properties/locales/en.json

```diff
@@ -67,6 +67,8 @@
   "expression.all.functions": "All Functions",
   "expression.min.label": "Min",
   "expression.max.label": "Max",
+  "toggle.on.label": "on",
```

Comment:
Should be On and Off.

Reply:
<img width="244" alt="image" src="https://user-images.githubusercontent.com/89609176/188948617-cbd24f6d-506e-4bfd-b389-0e838bdd18cb.png">